recipes/pyeo/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pyeo" %}
+{% set version = "0.65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/clcr/pyeo/archive/v0.65-alpha.tar.gz
+  sha256: c759301d6669e5e3a1ac182a9976436a2b1093ccd553ffaa2921901c115b55e3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+  skip: true  # [py<36]
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
+    - pip
+    - numpy 1.14.*
+  run:
+    - boto3
+    - botocore
+    - gdal
+    - joblib
+    - matplotlib
+    - pip
+    - pytest
+    - python
+    - requests
+    - scikit-image
+    - setuptools
+    - {{ pin_compatible('numpy') }}
+    - scikit-learn
+    - scipy
+    - python-fmask
+    - sentinelhub
+    - sentinelsat
+    - tenacity
+
+test:
+  imports:
+    - pyeo
+  requires:
+    - python
+  commands:
+    - python -c "import pyeo"
```

Comment:
This test is the same as the import test above. If it shouldn't be anything different (like run a test suite) it should be removed.